llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -18306,7 +18309,13 @@ NodeExtensionHelper::getSupportedFoldings(const SDNode *Root) {
   case RISCVISD::VFNMADD_VL:
   case RISCVISD::VFNMSUB_VL:
     Strategies.push_back(canFoldToVWWithSameExtension);
-    if (Root->getOpcode() == RISCVISD::VFMADD_VL)
+    if (Subtarget.hasStdExtZvfbfwma() &&
```

Comment:
Should Zvfbfa have higher priority here?

Reply:
If both Zvfbfa and Zvfbfwma are enabled at the same time, we shouldn't prevent the more capable Zvfbfa instructions from being used.